srcjs/shinyapp.js

```diff
@@ -503,6 +507,14 @@ var ShinyApp = function() {
     }
   });
 
+  addCustomMessageHandler('recalculating', function(message) {
+    if (message.hasOwnProperty('name')) return;
+    $('#' + message.name).trigger({
```

Comment:
I don't think we can assume that the element's `id` is the same as the output ID, can we? Do we make this assumption anywhere else? If not, I think ShinyApp has a dictionary between output ID and binding/element.